roles/ovirt-provider-ovn-driver/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+ovn_tunneling_network: ovirtmgmt
+ovn_central: ~
```

Comment:
Maybe it will be better to remove this varaible and do check if the variable `is defined` in the when clause latter.

Reply:
Done.